core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala

```diff
@@ -79,7 +79,7 @@ private[spark] class DiskBlockManager(
   private val shutdownHook = addShutdownHook()
 
   // If either of these features are enabled, we must change permissions on block manager
-  // directories and files to accomodate the shuffle service deleting files in a secure environment.
+  // directories and files to accommodate the shuffle service deleting files in a secure environment.
```

Comment:
See the test failure results. This line is too long now

`[error] /home/runner/work/spark/spark/core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala:82: File line length exceeds 100 characters`

Reply:
done